cdap-runtime-spi/src/main/java/io/cdap/cdap/runtime/spi/provisioner/ProgramRun.java

```diff
@@ -16,11 +16,16 @@
 
 package io.cdap.cdap.runtime.spi.provisioner;
 
+import io.cdap.cdap.runtime.spi.ProgramRunInfo;
+
 import java.util.Objects;
 
 /**
  * A program run.
+ *
+ * @deprecated Use {@link ProgramRunInfo} instead
```

Comment:
For deprecated methods, we usually add the release number from which the method/class is deprecated.
https://github.com/cdapio/cdap/blob/1d62163faaecb5b888f4bccd0fcf4a8d27bbd549/cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/validation/InvalidConfigPropertyException.java#L23

Reply:
fixed